### DISCUSSION TITLE
Im on an app using flutter and needs a floating window that can display over other app, im now using draggable but it can only display in the app, i have also added permission, i still dont know where the problem is

### DISCUSSION THREAD

**Discussion by hello-world0321** [community/community#125380] [Programming Help]
### Select Topic Area

Question

### Body

Im on an app using flutter and needs a floating window that can display over other app, im now using draggable but it can only display in the app, i have also added permission, i still dont know where the problem is.
can anyone give me some help? Thx

**Comment by Musaddiq625** [comment]
Hey @hello-world0321 
yes Draggable is to drag the widgets inside the app.
Try using this package: [floating](https://pub.dev/packages/floating)

This will allow you to run the widget/screen as overlay outside the app 🙌🏻
